Sources/StreamChat/Models/CurrentUser.swift

```diff
@@ -71,6 +71,7 @@ public class _CurrentChatUser<ExtraData: UserExtraData>: _ChatUser<ExtraData> {
         createdAt: Date = .init(),
         updatedAt: Date = .init(),
         lastActiveAt: Date? = nil,
+        teams: Set<TeamId> = [],
         extraData: ExtraData = .defaultValue,
         devices: [Device] = [],
         currentDevice: Device? = nil,
```

Comment:
The `init` of `CurrentUser` shouldn't be `public`. I think we should remove the default values from this init, too.

Reply:
Ah okay, I thought it was public for some reason. If it shouldn't be public, I'll remove it, and the default values too 👍

Reply:
✅

Reply:
@VojtaStavik Fixed here: https://github.com/GetStream/stream-chat-swift/pull/952/commits/a66c2900f01c37e693892a939c5e273e55eac4b5